containers/lightnode/whatisPalomaSection.tsx

```diff
@@ -49,11 +49,21 @@ const whatisPalomaSection = () => (
       <div>
         <h2>Paloma chain</h2>
         <span className="mt-20">
-          Today these state changes are commonly run by centralized, custom-designed, closed-source
-          programs, human-multisignature key arrangements, or low-end, multi-party computation
-          circuits. The complexity and richness of these approaches is limited by the lack of
-          engineering resources and developer-friendly environments that encourage sophisticated
-          state and logic-call development.
+        To solve this problem, Paloma aims to bring multichain, 
+        gas-optimized, MEV-protected, and scheduled, messages 
+        managed by a permissionless validator set that manages 
+        both chain security and message security.
+        
```

Comment:
Need to `<br />` tag for a line break in the text.
Since we need two line breaks, so add two <br> tags.
like this: 
```
<br />
<br />
```